### ISSUE TITLE
Bring xo Code Styling in to test-jstransformer

### ISSUE THREAD

**Issue by RobLoach** [jstransformers/test-jstransformer#28]
I'd like to re-visit this. While having xo as part of boilerplate is nice, it causes issues when we have to upgrade it individually. Takes a lot of manual work to update it.

Making the code style checks part of test-jstransformer would make the code style a lot easier to check for. Updates would be easier, and we wouldn't have a lot of that manual code in each package.json.

**Comment by calebeby** [comment]
Yeah, I can see the reasoning here. I approve of moving xo to test-jstransformer
